spec/Controller/Action/ListWishlistProductsActionSpec.php

```diff
@@ -78,17 +81,24 @@ function it_lists_wishlist_items(
         $wishlistContext->getWishlist($request)->willReturn($wishlist);
         $cartContext->getCart()->willReturn($cart);
         $wishlist->getWishlistProducts()->willReturn($wishlistProducts);
+
+        $commandsArray = new ArrayCollection();
```

Comment:
Why not extract it into private method or object mother: https://stackoverflow.com/questions/923319/what-is-an-objectmother